src/providers/zoom.js

```diff
@@ -0,0 +1,21 @@
+export default function Zoom(options) {
+  return {
+    id: "zoom",
+    name: "zoom",
+    type: "oauth",
+    version: "2.0",
+    params: { grant_type: "authorization_code" },
+    accessTokenUrl: "https://zoom.us/oauth/token",
+    authorizationUrl:
+      "https://zoom.us/oauth/authorize?response_type=code",
+    profileUrl: "https://api.zoom.us/v2/users/me",
+    profile(profile) {
+      return {
+        id: profile.id,
+        name: `${profile.first_name} ${profile.last_name}`,
+        email: profile.email,
+      }
+    },
```

Comment:
```suggestion
    },
    protection: ["pkce", "state"]
```

From the docs: https://marketplace.zoom.us/docs/guides/auth/oauth it seems that Zoom works with both state and pkce, so I think we should default to that maybe? Could you verify if that is the case? Also it would seem that this will have the same problem that is registered in #950. Have you actually been able to test this Provider? I am happy if it works with the current implementation, but I think #1846 will be useful in this case as well.